frontend/src/components/shared/StreamlitMarkdown/StreamlitMarkdown.tsx

```diff
@@ -230,11 +231,20 @@ class StreamlitMarkdown extends PureComponent<Props> {
   }
 }
 
+// Note: React markdown limits hrefs to specific protocols ('http', 'https',
+// 'mailto', 'tel') We are essentially allowing any URL (a data URL). It can
+// be considered a security flaw, but developers can choose to expose it.
+export function transformLinkUri(href: string): string {
```

Comment:
nit: this function might be moved above the `StreamlitMarkdown` class since it is only used there. Also, do we need the export keyword for this function?